core/src/test/java/io/questdb/test/cutlass/http/line/LineHttpFailureTest.java

```diff
@@ -427,7 +427,7 @@ public void testPutAndGetAreNotSupported() throws Exception {
                                     .send()
                     ) {
                         resp.await();
-                        TestUtils.assertEquals("400", resp.getStatusCode());
+                        TestUtils.assertEquals("404", resp.getStatusCode());
```

Comment:
we must send 405 response (method not allowed) to be correct

Reply:
changed in https://github.com/questdb/questdb/pull/5621/commits/643590a32ceaa422783452ec2a99f50cf265b2e5